### PR TITLE
Linux: Allow activating workspace with absolute path

### PR DESCRIPTION
**Pull request by JPTIZ**

### CODE REVIEW

File: ww [hunk 1]
```diff
@@ -15,8 +15,14 @@ activate() {
   if [ -z "$sourced" ]; then
     errorOut "use 'source ww $1'"
   fi
-  
-  workspace="${WW_DEFAULT_PATH}/${1}"
+
+  # If provided workspace is a absolute path, use it as so. If not, then make
```
Comment (prusse-martin) [inline ww:19]: But let you know this is not how it is done on windows.

On windows if you pass any path (absolute or relative) to a valid workspace it "activates" that workspace.

```
for %%i in (%WW_DEFAULT_VOLUMES%) do (
    if not exist %_WW_WORKSPACE%\%WW_PROJECTS_SUBDIR% (
        if exist %%i:\%_WW_WORKSPACE%\%WW_PROJECTS_SUBDIR% (
            set _WW_WORKSPACE=%%i:\%_WW_WORKSPACE%
        )
    )
)
...
REM Change WW_CURRENT_WORKSPACE to absolute PATH, if it is still relative
for /F "tokens=* delims=\" %%i in ("%_WW_WORKSPACE%") do set "WW_CURRENT_WORKSPACE=%%~fi"
```

The check is nested inside the for loop (not the best place for it to be, I agree on that).
The "valid"  check is "if arg1\Projects exist".

**Review by prusse-martin — APPROVED** [review]
We actually should make the scripts have the same behavior all around.
At least this way one became closer to the other.

File: ww (hunk 1 above)
Comment (JPTIZ) [reply ww]: Updated to mimic the Windows behaviour :)

Comment (JPTIZ) [reply ww]: *ping* could you re-review if the mimic was properly done?

File: ww [hunk 2]
```diff
@@ -16,7 +16,14 @@ activate() {
     errorOut "use 'source ww $1'"
   fi
 
-  workspace="${WW_DEFAULT_PATH}/${1}"
+  local workspace="${1}"
+
+  # Checks it provided workspace fits as a relative path.
+  # If not, then consider it an absolute path.
+  if [ -d "${WW_DEFAULT_PATH}/${workspace}" ]; then
```
Comment (prusse-martin) [inline ww:23]: I get that on linux there is not multiple workspace roots (like windows).

Not a problem.
I think having multiples roots not really nice =)

**Review by prusse-martin — APPROVED** [review]

**Pull request merged**